src/otx/recipe/detection/atss_mobilenetv2.yaml

```diff
@@ -1,65 +1,38 @@
-model:
-  class_path: otx.algo.detection.atss.MobileNetV2ATSS
-  init_args:
-    label_info: 1000
-
-    optimizer:
-      class_path: torch.optim.SGD
-      init_args:
-        lr: 0.004
-        momentum: 0.9
-        weight_decay: 0.0001
-
-    scheduler:
-      class_path: otx.core.schedulers.LinearWarmupSchedulerCallable
-      init_args:
-        num_warmup_steps: 3
-        main_scheduler_callable:
-          class_path: lightning.pytorch.cli.ReduceLROnPlateau
-          init_args:
-            mode: max
-            factor: 0.1
-            patience: 4
-            monitor: val/map_50
-
-engine:
-  task: DETECTION
-  device: auto
+engine: ../_base_/engine/detection.yaml
+model: ../_base_/model/detection.yaml
+data: ../_base_/data/detection.yaml
```

Comment:
I'm not sure if this is the right way to go, `data` pipeline has a lot of overlap and could be collected in common (This is actually not a good idea in my opinion.), but `engine` and `model` don't contain that much config, so I think it's okay to have a recipe for each model.
Also, this is the kind of deep configuration design we were trying to avoid in OTX 2.0.

A separate configuration for `engine`, `data`, and `model` was actually my first design proposal in OTX 2.0, (like Anomalib - https://github.com/openvinotoolkit/anomalib/tree/main/configs)
but it didn't go forward because there was a consensus to reduce it to a single configuration called recipe.

In my opinion, this change needs to be discussed. 
cc. @kprokofi @samet-akcay

Reply:
Thanks for the information, I didn't know the history.
During this progress, I thought it would be better to combine not only data but al oengine and model.
I'll revert engine and model.

Reply:
Reverted at e5104b0 and 6135260